va/va_vpp.h

```diff
@@ -1313,6 +1333,15 @@ typedef struct _VAProcFilterParameterBufferHVSNoiseReduction {
      *  Value 10 is the default value.
      */
     uint16_t            strength;
+    /**
+     *  \brief HVS Denoise Mode which controls denoise method.
+     *  It is a value in VAProcHVSDenoiseMode.
+     *  Value VAProcHVSDenoiseDefault means default mode;
```

Comment:
I don't think you need to repeat values descriptions here - you have them above within `VAProcHVSDenoiseMode`  declaration.